client/relay-chain-local/src/lib.rs

```diff
@@ -345,7 +343,7 @@ fn build_polkadot_full_node(
 }
 
 /// Builds a relay chain interface by constructing a full relay chain node
-pub fn build_relay_chain_interface(
+pub fn build_relay_chain_local(
 	polkadot_config: Configuration,
```

Comment:
Please correct me if I'm wrong, but I assume we want to eventually remove this so that the only supported way will be to link the two nodes by RPC, right? If so would it make sense to print out a `warn!` that this is deprecated and that people should link the nodes by RPC? (Or is it too early for that yet?)

Reply:
Yep, too early for now. There will be some follow-up PRs before I would actively advis users to switch to RPC. Might be worth adding a "experimental" flag or something to the command line 🤔

Reply:
Yeah, an experimental flag sounds good to me!

Reply:
Added `EXPERIMENTAL` to the cli flag